index.js

```diff
@@ -70,6 +70,27 @@ HtmlWebpackPlugin.prototype.emitHtml = function(compilation, htmlTemplateContent
   };
 };
 
+HtmlWebpackPlugin.prototype.getAssetPathFromModuleName = function(publicPath, modules) {
```

Comment:
It might be better to extract some if these functions into a `lib/` directory or something, as this file is getting a bit bulky, and there's no real need to export anything but the constructor and the `apply`-function